python/cudf/cudf/core/column/column.py

```diff
@@ -166,12 +166,6 @@ def equals(self, other):
             return True
         if other is None or len(self) != len(other):
             return False
-        if len(self) == 1:
```

Comment:
Is this an optimization?

Reply:
From this commit, https://github.com/rapidsai/cudf/commit/415dd9a2628d9d2c26a5a5344d9d68be53f273a0 looks like this one would be obsolete as we currently donot return ndarray incase of a mulitIndex:

```python
>>> df = cudf.DataFrame({'a': [1, 2, 3, 4], 'b': [7, np.NaN, 9, 10], 'c':[10, 11, 12, 12]})
>>> df.set_index(['a','b'])
         c
a b       
1 7     10
2 <NA>  11
3 9     12
4 10    12
>>> df.set_index(['a','b'])['c']
a  b   
1  7       10
2  <NA>    11
3  9       12
4  10      12
Name: c, dtype: int64
>>> df.set_index(['a','b']).index
MultiIndex(levels=[0    1
1    2
2    3
3    4
dtype: int64, 0     7
1     9
2    10
dtype: int64],
codes=   a  b
0  0  0
1  1 -1
2  2  1
3  3  2)
>>> df.set_index(['a','b']).index[0]
(1, 7)
>>> type(df.set_index(['a','b']).index[0])
<class 'tuple'>
```

Maybe @trxcllnt could be having more insight if I'm missing something here.